packages/design-system/src/styles/base/_link.scss

```diff
@@ -77,16 +69,10 @@ $link-underline-offset: auto !default;
   }
 
   /* stylelint-disable selector-max-specificity */
-  @if $ds-include-focus-styles {
-    &:focus,
-    &:focus:visited {
-      @include focus-styles;
-    }
-  } @else {
-    &:focus {
-      color: $link-inverse-hover-color;
-      text-decoration-color: $link-inverse-hover-color;
-    }
+  &:focus,
+  &:focus:visited {
+    @include focus-styles-link;
+    color: $color-base;
```

Comment:
Can't set the link color to white because the background is white so setting it to `color-base`

Reply:
Why not `$link-inverse-visited-color`?

Reply:
Hmmm, `$link-inverse-visited-color` uses `$color-muted-inverse` which is a really light grey and wouldn't pass contract on a white background - We add white background for the focus to ensure the link is visible on dark backgrounds in addition to the outline

Reply:
oh, I actually meant to copy and paste the non-inverse color variable

Reply:
`$link-visited-color`

Reply:
I updated to use `link-color` - this is only for focused elements so we wouldn't want to use the visited color here